zerver/migrations/0443_userpresence_new_table_schema.py

```diff
@@ -127,7 +127,12 @@ class Migration(migrations.Migration):
                 migrations.RenameModel(
                     old_name="UserPresence",
                     new_name="UserPresenceOld",
-                )
+                ),
+                migrations.RenameIndex(
+                    model_name="userpresenceold",
+                    old_name="zerver_userpresence_realm_id_timestamp_25f410da_idx",
+                    new_name="zerver_userpresenceold_realm_id_timestamp_52ef5fd3_idx",
```

Comment:
So my understanding here is that we cannot really guarantee that these `old_name` and `new_name` values here match the actual names that were involved - that's the reason for the fancy `rename_indexes_constraints` logic above that @alexmv  drafted up - on some servers the index names can be different somehow. But, since this is just in `state_operations` and also temporary (the whole `UserPresenceOld` table gets deleted 2 migrations later), I believe the potential mismatch of names does not pose a problem and can be ignored?

If my reasoning here makes sense to you, then this migration looks good to me. @andersk @timabbott

Reply:
That reasoning makes sense to me, merged, thanks @andersk!